libraries/redcore/api/soap/wsdl.php

```diff
@@ -279,102 +311,124 @@ public function generateWsdl()
 			// Read list
 			if (isset($this->webserviceXml->operations->read->list))
 			{
+				$filters = array();
+
+				if (isset($this->webserviceXml->operations->read->list->fields->field))
```

Comment:
I moved the function to the helper so I can use it here
Also added the other parameters I need to get a full definition of the filter (I need that for the wsdl, not just the filter name)